packages/driver/src/cy/commands/actions/type.coffee

```diff
@@ -170,16 +170,21 @@ module.exports = (Commands, Cypress, cy, state, config) ->
 
           return if not form.length
 
-          multipleInputsAndNoSubmitElements = (form) ->
-            inputs  = form.find("input")
+          multipleInputsBlockImplicitSubmissionAndNoSubmitElements = (form) ->
             submits = getDefaultButtons(form)
 
-            inputs.length > 1 and submits.length is 0
+            ## https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#implicit-submission
+            implicitSubmissionBlockingInputs = form.find("input").filter (__, input) ->
+              $input = $dom.wrap(input)
+              
+              $elements.blocksImplicitSubmission($input)
```

Comment:
This should ask a question because it's returning a boolean.

This is written as a verb but it doesn't actually **do** anything.

```js
$elements.isInputAndCausesImplicitFormSubmission($input)
```